config.go

```diff
@@ -171,7 +172,14 @@ type RemoteDaemonConfig struct {
 
 	// MacaroonDir is the directory that contains all the macaroon files
 	// required for the remote connection.
-	MacaroonDir string `long:"macaroondir" description:"The directory containing all lnd macaroons to use for the remote connection."`
+	MacaroonDir string `long:"macaroondir" description:"DEPRECATED: Use macaroonpath. The directory containing all lnd macaroons to use for the remote connection."`
+
+	// MacaroonPath is the path to the single macaroon that should be used
+	// instead of needing to specify the macaroon directory that contains
+	// all of lnd's macaroons. The specified macaroon MUST have all
+	// permissions that all the subservers use, otherwise permission errors
+	// will occur.
+	MacaroonPath string `long:"macaroonpath" description:"The full path to the single macaroon to use, either the admin.macaroon or a custom baked one. Cannot be specified at the same time as macaroondir. A custom macaroon must contain ALL permissions required for all subservers to work, otherwise permission errors will occur."`
```

Comment:
Are we going to add the bake command in the readme for a custom macaroon? I remember we discussed this a while back but don't remember exactly what we came up with. Could be nice to point people there if that's the plan.

Reply:
Yes, we'll probably do that eventually. But because things with macaroons will probably change again to support the "stateless remote mode", we'll probably look at it in that context again.